arch/cpu/cc26xx-cc13xx/rf-core/rf-core.h

```diff
@@ -263,12 +274,74 @@ typedef struct rf_core_primary_mode_s {
 /* Radio timer register */
 #define RATCNT  0x00000004
 /*---------------------------------------------------------------------------*/
-/* Buffer full flag */
-extern volatile bool rx_is_full;
+/* Special value returned by CMD_IEEE_CCA_REQ when an RSSI is not available */
+#define RF_CORE_CMD_CCA_REQ_RSSI_UNKNOWN     -128
+
+/* Used for the return value of channel_clear */
+#define RF_CORE_CCA_CLEAR                       1
+#define RF_CORE_CCA_BUSY                        0
+
+/* Used as an error return value for get_cca_info */
+#define RF_CORE_GET_CCA_INFO_ERROR           0xFF
+
+/*
+ * Values of the individual bits of the ccaInfo field in CMD_IEEE_CCA_REQ's
+ * status struct
+ */
+#define RF_CORE_CMD_CCA_REQ_CCA_STATE_IDLE      0 /* 00 */
+#define RF_CORE_CMD_CCA_REQ_CCA_STATE_BUSY      1 /* 01 */
+#define RF_CORE_CMD_CCA_REQ_CCA_STATE_INVALID   2 /* 10 */
+
+#define RF_CORE_CMD_CCA_REQ_CCA_CORR_IDLE       (0 << 4)
+#define RF_CORE_CMD_CCA_REQ_CCA_CORR_BUSY       (1 << 4)
+#define RF_CORE_CMD_CCA_REQ_CCA_CORR_INVALID    (3 << 4)
+#define RF_CORE_CMD_CCA_REQ_CCA_CORR_MASK       (3 << 4)
+
+#define RF_CORE_CMD_CCA_REQ_CCA_SYNC_BUSY       (1 << 6)
+/*---------------------------------------------------------------------------*/
+#define RF_CORE_RX_BUF_INCLUDE_CRC 0
+#define RF_CORE_RX_BUF_INCLUDE_RSSI 1
+#define RF_CORE_RX_BUF_INCLUDE_CORR 1
+#define RF_CORE_RX_BUF_INCLUDE_TIMESTAMP 1
+/*---------------------------------------------------------------------------*/
+/* How long to wait for an ongoing ACK TX to finish before starting frame TX */
+#define RF_CORE_TX_TIMEOUT       (RTIMER_SECOND >> 11)
+
+/* How long to wait for the RF to enter RX in rf_cmd_ieee_rx */
+#define RF_CORE_ENTER_RX_TIMEOUT (RTIMER_SECOND >> 10)
+
+/* How long to wait for the RF to react on CMD_ABORT: around 1 msec */
+#define RF_CORE_TURN_OFF_TIMEOUT (RTIMER_SECOND >> 10)
+
+/* How long to wait for the RF to finish TX of a packet or an ACK */
+#define RF_CORE_TX_FINISH_TIMEOUT (RTIMER_SECOND >> 7)
+
+#define RF_CORE_LIMITED_BUSYWAIT(cond, timeout) do {                 \
```

Comment:
please replace with new busy-wait routines when https://github.com/contiki-ng/contiki-ng/pull/562 is merged